config/checker/com.intellij.astub

```diff
@@ -283,9 +283,14 @@ package com.intellij.psi.search.scope.packageSet;
 class CollectionComboBoxModel<T> {
   // This method is annotated as `org.jetbrains.annotations.Nullable` already,
   // but for some reason Checker unsoundly assumes that it's `NonNull` instead.
+  @SafeEffect
   @Nullable T getSelected();
 }
 
+class MutableCollectionComboBoxModel<T> {
+  @SafeEffect T getSelected();
```

Comment:
Nit: move `@SafeEffect` to a separate line since it applies to the method as a whole, not to its return type (unlike e.g. `@Nullable`)